src/appleseed.python/bindaov.cpp

```diff
@@ -91,6 +94,21 @@ namespace
 
         return channels;
     }
+
+    Image& get_crypto_image(AOV* aov)
```

Comment:
It should be possible to make this a const pointer.

Reply:
The static cast doesn’t work if it’s a const pointer

Reply:
Never mind, I wasn't doing the cast correctly